pkg/transformer/openshift/openshift.go

```diff
@@ -161,7 +145,17 @@ func getAbsBuildContext(context string) (string, error) {
 
 // initImageStream initialize ImageStream object
 func (o *OpenShift) initImageStream(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) *imageapi.ImageStream {
+	// Use "name" by default (if service.Image isn't set)
+	image := name
 	tag := getImageTag(service.Image)
+	// Use image if it was set
+	if service.Image != "" {
+		image = service.Image
+	}
+	// Add a tag if not present
+	if !strings.Contains(image, ":") {
+		image = image + ":" + tag
+	}
```

Comment:
@cdrage right now, irrespective if image is set or not, or tag is present or not, `image := name` is being set above and overwritten later.
Should we handle this using switch case, and put `image := name` as the default case?